### PR TITLE
added funny comments - Fisnished Modifiy employee story

### PR DESCRIPTION
**Pull request by anillibra**
review pls. ?

### CODE REVIEW

File: employee.ts [hunk 1]
```diff
@@ -10,4 +10,10 @@ export class Employee {
     add(): void {
         this.employees.push({ _id: 1, name: "Parik" });
     }
+
```
Comment (shimanshu) [inline employee.ts:13]: Review Started

File: employee.ts [hunk 2]
```diff
@@ -10,4 +10,10 @@ export class Employee {
...
+    // Added modify function for employees  
+    modify(): void {
```
Comment (shimanshu) [inline employee.ts]: rename modify() to mofifyEmployee()

**Review by shimanshu — CHANGES_REQUESTED** [review]
Please make all suggested change

File: employee.ts (hunk 2 above)
Comment (anillibra) [reply employee.ts]: ok done this change

File: employee.ts (hunk 1 above)
Comment (anillibra) [reply employee.ts:13]: finished

File: employee.ts [hunk 3]
```diff
@@ -10,4 +10,10 @@ export class Employee {
     add(): void {
         this.employees.push({ _id: 1, name: "Parik" });
     }
+
+    // Added modify function for employees  
+    modifyEmployee(): void {
+        this.employees.splice(0, 1, { _id: 1, name: "Parik Sheetle" });
+        // this employee can't be modified
```
Comment (shimanshu) [inline employee.ts:17]: Change comment to employee can be modify

File: employee.ts [hunk 4]
```diff
@@ -10,4 +10,10 @@ export class Employee {
     add(): void {
         this.employees.push({ _id: 1, name: "Parik" });
     }
+
+    // Added modify function for employees
```
Comment (shimanshu) [inline employee.ts:14]: Change comment added modify for employee

**Review by shimanshu — CHANGES_REQUESTED** [review]
Please change

File: employee.ts (hunk 3 above)
Comment (anillibra) [reply employee.ts:17]: ok

File: employee.ts (hunk 4 above)
Comment (anillibra) [reply employee.ts:14]: done

File: employee.ts (hunk 3 above)
Comment (shimanshu) [reply employee.ts:17]: change void to string

**Review by shimanshu — APPROVED** [review]

**Pull request merged**